hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/5412-during-partition-fullUrl-not-shown-in-response.yaml

```
@@ -0,0 +1,5 @@
+---
+type: fix
+issue: 5412
+title: "Previously when partitions were enabled and a bundle was posted the response.link would display the
+partition name twice, now it only displays once."
```

Previously when partitions were enabled and a bundle was posted the response.link would display the
```suggestion
title: "Previously, with Partitioning enabled, submitting a bundle request would return response with the partition name displayed twice in `response.link` property. This has been fixed."
```